scripts/start.js

```diff
@@ -170,6 +171,20 @@ function openBrowser(port, protocol) {
   opn(protocol + '://localhost:' + port + '/');
 }
 
+function checkRequiredFiles() {
+  var filesPathToCheck = [paths.appHtml, paths.appIndexJs, paths.appFavico];
+  filesPathToCheck.forEach(function(filePath) {
+    fs.access(filePath, fs.constants.F_OK, function(err) {
```

Comment:
This is asynchronous so you are going to have race conditions. The code below will execute before these callbacks.

Instead, we should do a synchronous access here.